VMBackup/main/guestsnapshotter.py

```diff
@@ -95,7 +95,8 @@ def snapshot(self, sasuri, sasuri_index, settings, meta_data, snapshot_result_er
                         value = meta['Value']
                         headers["x-ms-meta-" + key] = value
                 if(CommonVariables.isSnapshotTtlEnabled in settings and settings[CommonVariables.isSnapshotTtlEnabled]):
-                    headers[CommonVariables.snapshotTtlHeader] = '0'
+                    self.logger.log("Not passing the TTL header via Guest path though it is enabled")
+                    #headers[CommonVariables.snapshotTtlHeader] = '0'
```

Comment:
In general, let's not have commented code in our codebase. The only times I've found commented code to be useful is when it somehow helps local machine setup/testing.

For any other context, the code should be either there or not. Not in between.